Formula/l/llvm@14.rb

```diff
@@ -32,10 +32,9 @@ class LlvmAT14 < Formula
   depends_on "cmake" => :build
   # sanitizer_mac.cpp:621:15: error: constexpr function never produces a constant expression [-Winvalid-constexpr]
   # constexpr u16 GetOSMajorKernelOffset() {
-  depends_on maximum_macos: [:sonoma, :build]
-  depends_on "python@3.12" => :build
+  depends_on maximum_macos: [:ventura, :build]
```

Comment:
now this error also happens with sonoma runner

Reply:
Probably because of Xcode update